src/validation.cpp

```diff
@@ -5011,10 +5071,10 @@ bool LoadMempool(CTxMemPool& pool)
             TxValidationState state;
```

Comment:
Nit: Is `state` still needed here?

Reply:
good catch, removed 👍 
(btw, how did you comment on a non-diff line? 😱 )